hw/top_earlgrey/syn/chip_earlgrey_asic.sdc

```diff
@@ -161,6 +161,36 @@ set JTAG_TCK_PERIOD [expr $JTAG_TCK_TARGET_PERIOD*$CLK_PERIOD_FACTOR]
 create_clock -name JTAG_TCK -period $JTAG_TCK_PERIOD [get_ports $JTAG_CLK_PIN]
 #set_ideal_network [get_ports $JTAG_CLK_PIN]
 set_clock_uncertainty ${SETUP_CLOCK_UNCERTAINTY} [get_clocks JTAG_TCK]
+set_propagated_clock JTAG_TCK
```

Comment:
Should we try setting `JTAG_TCK_TARGET_PERIOD` top 30MHz to maximize performance?